docs/school-check-data.md

```diff
@@ -0,0 +1,65 @@
+# Generating School Check Data
+
+**NB: This process involves accessing the production console. You will need an
+approved PIM request, and a second developer to observe during this.**
+
+As part of verifying a teacher's claim service operators will send emails to the
+relevant schools to check their employment status. Generating a list of claims
+which need this email sending is a developer task, documented below.
+
+## Process
+
+1. Visit the
+   [School Check Email - Sent Claims spreadsheet](https://docs.google.com/spreadsheets/d/1mxMND-SqOjK7yyYp0EaEJOOCUYFkqLJ1ZIwPLuSVKfM/edit#gid=1926551388)
+
+2. On the second sheet, copy the contents of cell A1. This should be a
+   comma-delimited list of all the claim references which have already been
+   checked, for example `A1B2C345,D6E7F890,G1H2I345...`. Check to make sure
+   there are no trailing commas
+
+3. Have a second developer join you too observe, and connect to the production
+   console using
+
+   ```
+   bin/azure-console production
+   ```
+
+4. Activate the Rails console in sandbox mode with
+
+   ```
+   bundle exec rails console --sandbox
+   ```
+
+5. Create a new instance of `Claim::SchoolCheckEmailDataExport` with
+
+   ```
+   exporter = Claim::SchoolCheckEmailDataExport.new("A1B2C345,D6E7F890,G1H2I345...")
+   ```
+
+   substituting the claim references for the string you copied from the
+   spreadsheet
+
+6. Generate the CSV data of claims needing checking by running
+
+   ```
+   exporter.csv_string
```

Comment:
Using `puts` here will directly output the string to the terminal STDOUT making step 7 redundant.

```suggestion
   puts exporter.csv_string
```